src/components/card/index.tsx

```diff
@@ -69,7 +69,7 @@ export function ReviewCard(props) {
 					{review.userId}
 					<span className="text-[#87868C]">
 						&#x2022;{" "}
-						{`${date.getDate()}/${date.getMonth()}/${date.getFullYear()}`}
+						{`${date.getDate()}/${date.getMonth()+1}/${date.getFullYear()}`}
```

Comment:
@piyush-bitpack does `date.getMonth()` always return a number? Even if it fails for any reason?